CalibTracker/SiStripLorentzAngle/interface/TIBorTOBparameters.h

```diff
@@ -0,0 +1,21 @@
+#ifndef TIBORTOBPARAMETERS_H
+#define TIBORTOBPARAMETERS_H
+
+#include "DataFormats/SiStripDetId/interface/SiStripDetId.h"
+
+// WARNING: This header file is created on the assumption
+// that the subDetector Id (TIB or TOB) is already known.
+// It is used inside CalibTracker/SiStripLorentzAngle only.
```

Comment:
+1, maybe also specify in which package/subpackage the TIBDetId are, we're going to deprecate them so you want to ease life of people possibly digging history some day...